src/Sarif.Viewer.VisualStudio/FileWatcher/FileWatcher.cs

```diff
@@ -96,8 +146,13 @@ private void SarifLogFile_Changed(object sender, FileSystemEventArgs e)
         {
             try
             {
-                // to avoid trigger changed event multiple times in a short time period.
+                // some processes update files using complex mechanisim, may cause file changed
```

Comment:
>mechanisim [](start = 61, length = 10)

mechanism

Reply:
corrected

---
In reply to: [602309794](https://github.com/microsoft/sarif-visualstudio-extension/pull/382#discussion_r602309794) [](ancestors = 602309794)